src/views/Scenario/Scenario.js

```diff
@@ -286,8 +323,19 @@ const Scenario = () => {
             </Card>
           </Grid>
           <Grid item xs={12}>
-            <Card component={Paper}>
-              <CardContent>
+            <Card component={Paper} sx={{ p: 0 }}>
```

Comment:
I think in fact, it's better to move the card management in ScenarioPowerBiReport component instead of manage this in Scenario view.